css/style.css

```diff
@@ -1,3 +1,522 @@
+}
+
+.header__menu {
+  position: relative;
+  width: 1170px;
+  height: 35px;
+  align-self: center;
+  font-size: 15px;
+  line-height: 11px;
+  text-align: justify;
+  text-transform: uppercase;
+  color: black;
+  font-weight: bold;
+}
+
+.menu__link {
+  color: black;
+  text-decoration: none;
+  align-self: center;
+  transition: text-decoration, font-size, letter-spacing 1s ease;
+}
+
+.menu__link:hover {
+  text-decoration: underline;
+  font-size: 20px;
+  letter-spacing: 2px;
+}
+
+.menu__left {
+  position: absolute;
+  display: flex;
+  width: 500px;
+  height: 30px;
+  justify-content: space-around;
+  align-content: center;
+  left: 13px;
+  bottom: 21px;
+  margin-right: 20px;
+}
+
+.menu__right {
+  position: absolute;
+  display: flex;
+  width: 535px;
+  height: 30px;
+  justify-content: space-around;
+  align-content: center;
+  right: 0px;
+  bottom: 21px;
+}
+
+.menu__search {
+  width: 216px;
+  height: 18px;
+  border-bottom: 1px solid #000000;
+  align-self: flex-end;
+  cursor: text;
+  position: relative;
+}
+
+.search__img {
+  position: absolute;
+  right: 0%;
+  cursor: pointer;
+  width: 16px;
+  height: 17px;
+  bottom: 5px;
+  transition: transform 1s linear;
+}
+
+.search__img:hover {
+  transform: scale(1.2, 1.2);
+}
+
+.cart__count {
+  display: inline-block;
+  width: 12px;
+  height: 12px;
+  color: white;
+  padding-top: 2px;
+  font-size: 12px;
+  text-align: center;
+  background: black;
+  border: 3px solid #000000;
+  border-radius: 50%;
+  margin-right: 45px;
+}
+
+.menu__flag {
+  display: inline-block;
+  position: absolute;
+  width: 23px;
+  height: 17px;
+  cursor: pointer;
+  align-self: center;
+  right: 15px;
+  transition: transform 1s linear;
+}
+
+.menu__flag:hover {
+  transform: scale(1.2, 1.2);
+}
+
+.new-collection__goto {
+  display: flex;
+  justify-content: center;
+  position: relative;
+  width: 1230px;
+  margin: 0 auto 90px auto;
+}
+
+.new-collection__goto .goto__img {
+  width: 100%;
+}
+
+.new-collection__goto .goto__title {
+  position: absolute;
+  display: inline-block;
+  color: white;
+  font-size: 40px;
+  text-transform: uppercase;
+  text-align: justify;
+  white-space: pre-line;
+  left: 63px;
+  align-self: center;
+  line-height: 1.2;
+}
+
+.goto__shop {
+  font-size: 15px;
+}
+
+.goto__arrow {
+  width: 70px;
+  cursor: pointer;
+  transition: transform 1s linear;
+}
+
+.goto__arrow:hover {
+  transform: scale(1.05, 1.05);
+}
+
+.products__title {
+  font-size: 30px;
+  margin: 0 auto 15px auto;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.products__list {
+  display: flex;
+  flex-direction: row;
+  width: 1200px;
+  height: 350px;
+  justify-content: space-between;
+  margin: 0 auto;
+  text-align: center;
+}
+
+/*rasta*/
+
+.products__product-rasta {
+  align-self: flex-start;
+  text-align: left;
+  position: relative;
+  width: 280px;
+  height: 340px;
+}
+
+.products__product-rasta::after {
+  content: "";
+  border-right: 1px solid #CCCCCC;
```

Comment:
Codacy found an issue: [Expected "#CCCCCC" to be "#cccccc" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/eyurodin_axion/pullRequest?prid=5738041)